common/repositories/base/base_variables_repository.h

```diff
@@ -135,7 +135,7 @@ class BaseVariablesRepository {
 			e.varname     = row[1] ? row[1] : "";
 			e.value       = row[2] ? row[2] : "";
 			e.information = row[3] ? row[3] : "";
-			e.ts          = row[4] ? row[4] : std::time(nullptr);
+			e.ts          = row[4] ? reinterpret_cast<time_t>(row[4]) : std::time(nullptr);
```

Comment:
ts is a string object in the struct

Reply:
By default this had issues is why I added the `reinterpret_cast` to `variables` and `account_ip`.
![image](https://github.com/EQEmu/Server/assets/89047260/841445c2-6932-4d33-898b-3bb0276bedd1)
![image](https://github.com/EQEmu/Server/assets/89047260/c5e92648-d356-4df5-bf68-d5316aa7ade0)

Reply:
We weren't using these repositories before so I guess it was never an issue.

Reply:
We need to implement the type on the struct `time_t` - for datetime we do translations between unix timestamp because that's what `time_t` represents and then we `FROM_UNIXTIME` when we save. See `datetime` handling in repository generator